### PR TITLE
Android/iOS SDK page adding mobile CSTG info

### PR DESCRIPTION
**Pull request by sunnywu**
1. This PR won't be merged until @genwhittTTD has converted the Word mobile integration guides into markdown pages in this repo and replace the Word doc links in this PR with actual doc site links
2. I am proposing to have a tvOS section in the UID2 SDK for iOS reference guide to mention it does support tvOS. Also, in android guide, to mention it support other non-mobile android devices too.
3. I have modified the description 
_Android/iOS SDK Reference for publishers who want to establish client identity using UID2_
to 
_Android/iOS SDK reference for publishers who want to generate or establish client identity using UID2_
as we are adding token generation functionality to both SDKs

### CODE REVIEW

File: docs/sdks/uid2-sdk-ref-android.md [hunk 1]
```diff
@@ -110,13 +110,16 @@ To install with Maven, add the SDK as a dependency in the `pom.xml` file:
...
+  <version>1.0.0</version> 
 </dependency> 
 ```
 
 ## Usage Guidelines
 
-The initial UID2 Identity *must* be generated by the implementing application and then passed into the UID2 SDK. Because of security requirements, the UID2 Mobile SDKs cannot create new UID2 identities.
+There are two ways to establish an initial UID2 Identity:
+
+1. Generate the UID2 identity using a DII - email (hash) or phone no (hash). For integration instructions, refer to [Client-Server Integration Guide for Mobile](https://ttdcorp-my.sharepoint.com/:w:/g/personal/sunny_wu_thetradedesk_com/EX2dEa2NAcRPs3Pfchwua40ByY0g0DASgLybKJs11TLHqw?e=vxRjFa).
```
Comment (sunnywu) [inline docs/sdks/uid2-sdk-ref-android.md:121]: will replace these word doc links to proper unifiedid.com pages before merging

File: docs/sdks/uid2-sdk-ref-ios.md [hunk 2]
```diff
@@ -93,19 +93,27 @@ Add the following dependency to Package.swift:
...
 dependencies: [
-.package(url: \"https://github.com/IABTechLab/uid2-ios-sdk.git\", from: "0.3.0"),
+.package(url: \"https://github.com/IABTechLab/uid2-ios-sdk.git\", from: "1.0.0"),
 ]
 ```
 
 ### Installing with Xcode
 
-Add the following dependency via Xcode:
+Add the following dependency via Xcode Package Dependencies:
```
Comment (sunnywu) [inline docs/sdks/uid2-sdk-ref-ios.md:102]: @dcaunt i want to replace the screenshot as shown on https://unifiedid.com/docs/sdks/uid2-sdk-ref-ios#installing-with-xcode as everytime we make a new release we need to create a new screenshot. 

Please let me know if this description and table below makes the same info clear or not, and if not, how we could improve it. Thanks

File: docs/sdks/uid2-sdk-ref-ios.md [hunk 3]
```diff
@@ -27,22 +27,27 @@ import Link from '@docusaurus/Link';
...
+| Purpose                                                                                                                                                                                                                 | Product/Documentation |
+|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------| :--- |
+| To use the Google Mobile Ads (GMA) SDK to send [UID2 tokens](../ref-info/glossary-uid.md#gl-uid2-token) as [secure signals](https://support.google.com/admob/answer/11556288) in ad requests from iOS/tvOS apps         | [UID2 GMA Plugin for iOS Integration Guide](../guides/mobile-plugin-gma-ios.md) |
+| To use the Google Interactive Media Ads SDK for iOS to send [UID2 tokens](../ref-info/glossary-uid.md#gl-uid2-token) as [secure signals](https://support.google.com/admob/answer/11556288) in ad requests from iOS/tvOS apps | [UID2 IMA Plugin for iOS Integration Guide](../guides/mobile-plugin-ima-ios.md) |
+
+
+## tvOS Support
```
Comment (sunnywu) [inline docs/sdks/uid2-sdk-ref-ios.md:40]: if we do not want to mention tvOS anywhere else, we should at least keep this section

File: docs/sdks/uid2-sdk-ref-ios.md (hunk 2 above)
Comment (sunnywu) [reply docs/sdks/uid2-sdk-ref-ios.md]: thanks @dcaunt i have corrected it per your suggestion and i reworded the top description from:

_Add the following dependency via Xcode Package Dependencies:_
to 
_Add the following entry in your application's Package Dependencies:_

which may be clearer

File: docs/sdks/uid2-sdk-ref-ios.md (hunk 3 above)
Comment (sunnywu) [reply docs/sdks/uid2-sdk-ref-ios.md:41]: decided to add this section and do not mention tvOS in other pages

File: docs/sdks/uid2-sdk-ref-ios.md [hunk 4]
```diff
@@ -120,15 +135,21 @@ This section includes the functions and variables that are part of the UID2Manag
...
 - [refreshIdentity()](#refreshidentity)
 - [getAdvertisingToken()](#getadvertisingtoken)
 - [setAutomaticRefreshEnabled()](#setautomaticrefreshenabled)
 
+#### generateIdentity()
+
+Generate a UID2 Identity using a <Link href="../ref-info/glossary-uid#gl-dii">Directly identifying information (DII)</Link>. Refer to
+[Client-Server Integration Guide for Mobile](https://ttdcorp-my.sharepoint.com/:w:/g/personal/sunny_wu_thetradedesk_com/EX2dEa2NAcRPs3Pfchwua40ByY0g0DASgLybKJs11TLHqw?e=vxRjFa)
+
 #### setIdentity()
```
Comment (genwhittTTD) [inline docs/sdks/uid2-sdk-ref-ios.md:150]: Hi @sunnywu . For iOS, generateIdentity links to Client-Server guide and setIdentity links to Client-Side guide. Is this the wrong way around? I believe it's the opposite for Android.

File: docs/sdks/uid2-sdk-ref-ios.md [hunk 5]
```diff
@@ -120,15 +135,21 @@ This section includes the functions and variables that are part of the UID2Manag
...
 
-Sets the UID2 Identity to be managed by the SDK.
+Sets a UID2 Identity created from backend server to be managed by the SDK. Refer to [Client-Side Integration Guide for Mobile/Android](https://ttdcorp-my.sharepoint.com/:w:/g/personal/sunny_wu_thetradedesk_com/EV4SjPzJjCZCig_kI8lGjJYBrLLhiiNh3mHq75pAtSs2qQ?e=2F7aMb)
```
[the remark this answers is not in the document]
Comment (sunnywu) [reply docs/sdks/uid2-sdk-ref-ios.md:152]: yes it's wrong way around please fix it :)

File: docs/sdks/uid2-sdk-ref-android.md [hunk 6]
```diff
@@ -131,15 +139,21 @@ This section includes the functions and variables that are part of the UID2Manag
...
 
+#### generateIdentity()
+
+Generate a UID2 Identity using a <Link href="../ref-info/glossary-uid#gl-dii">Directly identifying information (DII)</Link>. Refer to [Client-Side Integration Guide for Mobile/Android](https://ttdcorp-my.sharepoint.com/:w:/g/personal/sunny_wu_thetradedesk_com/EV4SjPzJjCZCig_kI8lGjJYBrLLhiiNh3mHq75pAtSs2qQ?e=2F7aMb)
+
+
 #### setIdentity()
 
-Sets the UID2 Identity to be managed by the SDK.
+Sets a UID2 Identity created from backend server to be managed by the SDK. Refer to [Client-Server Integration Guide for Mobile](https://ttdcorp-my.sharepoint.com/:w:/g/personal/sunny_wu_thetradedesk_com/EX2dEa2NAcRPs3Pfchwua40ByY0g0DASgLybKJs11TLHqw?e=vxRjFa)
```
[the remark this answers is not in the document]
Comment (sunnywu) [reply docs/sdks/uid2-sdk-ref-android.md:156]: yes ios has wrong way around

[the remark this answers is not in the document]
Comment (genwhittTTD) [reply docs/sdks/uid2-sdk-ref-android.md:156]: Updated in mine but please cross-check. Thx!

**Comment by sunnywu** [comment]
https://github.com/IABTechLab/uid2docs/pull/557 already included the changes in this PR. closing

**Pull request closed without merging**